src/Plane/PlaneFileGlue.cpp

```diff
@@ -172,10 +181,10 @@ PlaneGlue::Write(const Plane &plane, KeyValueFileWriter &writer)
   FormatPolarShape(plane.polar_shape, tmp.buffer(), tmp.capacity());
   writer.Write("PolarInformation", tmp);
 
-  tmp.Format("%f", (double)plane.reference_mass);
+  tmp.Format("%f", (double)plane.polar_shape.reference_mass);
   writer.Write("PolarReferenceMass", tmp);
-  tmp.Format("%f", (double)plane.dry_mass);
-  writer.Write("PolarDryMass", tmp);
```

Comment:
I think it would be better to write both for some time, to keep some compatibility with old versions, just in case somebody downgrades XCSoar for some reason. It should be accompanied with a good code comment.

Reply:
Good hint.

Reply:
Old dry_mass kept in file and structure, keeping it up to date just for a potential downgrade of the xcs.

Reply:
There's no code comment explanining the deprecation. Writing one takes less than a minute, and will save the next guy an hour.

Reply:
Added one.